git-multimail/git_multimail.py

```diff
@@ -926,7 +926,7 @@ def generate_email_intro(self, html_escape_val=False):
 
         raise NotImplementedError()
 
-    def generate_email_body(self):
+    def generate_email_body(self, push):
```

Comment:
The change looks good, but as much as possible try to follow the following for the commit message:

```
1 short line (ideally <50 characters, and anyway <80) describing what is done, followed by a blank line

Text describing why the change is good, if needed.
```